test/multi-select-list.test.js

```diff
@@ -51,7 +54,10 @@ describe('multi-select-list', () => {
 			await expect(el).to.be.accessible();
 		});
 
-		it('should pass all aXe tests when collapsible', async() => {
+		it('should pass all aXe tests when collapsible', async function() {
+			if (isSafariBrowser) {
+				this.skip();
+			}
```

Comment:
My preference is to keep something like this as tucked away as possible, but it's fine either way.
```suggestion
			isSafariBrowser && this.skip();
```